<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditLeft">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRight">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="lineEditResult">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: purple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReverseLeft">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSwap">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Swap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReverseRight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConcatLeft">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Left + Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReverseAll">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConcatRight">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Right + Left</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
